### DISCUSSION TITLE
Multi tenant possibility

### DISCUSSION THREAD

**Discussion by rnijs** [flowsforapex/apex-flowsforapex#594] [Q&A]
Hi all,

I'm just wondering when you want to use this "flows for apex" application for multiple customers, is there any possibility of defining this as a multi-tenant application? on user level?

Example: When logging in to the application, you do not see all workflows defined, but only for a particular category (customer).

Is this possible in the latest version? 
Thanks for the information.

Regards,
Ronny

**Comment by commi235** [comment]
Hi Ronny,

Currently this is not possible without modifying the engine app and other code on your own.
However we have this feature in our backlog, but are currently not having enough resources to get this feature in.

I would be happy to discuss your requirements, please contact me via [moritz.klein@mt-itsolutions.com](mailto:moritz.klein@mt-itsolutions.com).
